docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using the *Create project* wizard.
+
+.Procedure
+
+. In the {WebName}, click *Projects*.
+. Click *Create project*.
++
+.Project details screen
+image::web-add-project_51.png[Project details screen]
+
+. Enter a unique name for the project, an optional description, and click *Next*.
++
+.Applications screen
+image::web-add-apps_51.png[Applications screen]
++
+. To upload applications, click *Browse*, select the application files you want to upload, and click *Close*.
++
+Uploading applications stores them directly in the {ProductShortName} server.
+
+. To register a server path, click the *Server path* tab and enter the *Server-side path* of the application in the field.
++
+Registering the server path of the file tells the {ProductShortName} server the location of the application. This method is useful if you expect to update the application because it ensures that the {ProductShortName} server always uses the latest version of the application.
+
+ . Click *Next*.
++
+.Transformation target screen
+image::web-configure-analysis_51.png[Transformation target screen]
+
+. Click the desired transformation targets.
+. Click *Next*.
++
+You can specify which Java application packages to include or exclude on the *Select applications* screen.
++
+The list on the left contains all of the identified packages in the project. The list on the right contains only the packages that are to be analyzed.
+
+. Select the packages you want to include or exclude and click *>* or *<*.
+. Click *Next*.
++
+.Custom rules screen
+image::web-custom-rules-project_51.png[Custom rules screen]
++
+You can add custom rules to use in each project by uploading the custom rules or registering a server path that contains the rules.
+
+. If you want to add a rule, click *Add rule*.
+. To upload rules, click *Browse*, select a rule set file (`\*.mta.xml` or `*.windup.xml`), and click *Close*.
++
+Uploading rules stores them directly in the {ProductShortName} server.
++
+. To register a server path, click the *Server path* tab, enter the *Rules path* of the rules file in the field, and click *Save*.
++
+Registering the server path of a file tells the {ProductShortName} server the location of the rules. This method is useful if you expect to update the rules because it ensures that the {ProductShortName} server always uses the latest version of the rules.
++
+[NOTE]
+====
+For information on writing custom {ProductShortName} rules, see the {ProductShortName} link:{ProductDocRulesGuideURL}[_{RulesDevBookName}_].
+====
++
+. Click *Next*.
++
+.Custom labels screen
```

Comment:
My comments about custom rules also apply to custom labels.

Get rid of the screen shot, bring the text in line with the above.
Is there a guide for creating custom labels that can be referenced? Not necessary, just curious.